CRM/Multicompanyaccounting/Hook/BuildForm/FinancialAccount.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class CRM_Multicompanyaccounting_Hook_BuildForm_FinancialAccount {
+
+  private $form;
+
+  public function __construct($form) {
+    $this->form = $form;
+  }
+
+  public function run() {
+    $this->editOwnerField();
```

Comment:
the name `editOwnerField` does not mean much or reflect what the method does, a more accurate name might be:

`restrictOwnerFieldToCompanyOrgnisations`